views/templates/block/checklist.twig

```diff
@@ -145,30 +145,92 @@
                         {% endif %}
                     </td>
                 </tr>
+                {% if not checkPhpVersionCompatibility %}
+                  <tr>
+                    <td>{{ 'Your current PHP version isn\'t compatible with your PrestaShop version.'|trans }}</td>
```

Comment:
Notice that new wordings need to be added here: https://github.com/PrestaShop/autoupgrade/blob/dev/translations/en.php

See [how to create dictionary files](https://devdocs.prestashop.com/1.7/modules/creation/module-translation/classic-system/#creating-translation-dictionary-files)

Reply:
@eternoendless maybe it would be good to create a CLI command for it? 🤔

Reply:
Of course. So many things to do, so little time!

Reply:
@kpodemski Please create an issue ;) (or create a PR ;) ;))

Reply:
@eternoendless Is it normal that Translations doesn't detect change in twig files ?

Reply:
@eternoendless I think i have found the problem : #443

Reply:
The upgrade module is special, it doesn't use the new translation system but reroutes straight to the legacy one, for cross version compatibility.

Reply:
I'm sorry @eternoendless but  with the class system of tranlation, new translations string in twig files are not detected. May be I have done an error? 

With [this process](https://devdocs.prestashop.com/1.7/modules/creation/module-translation/classic-system/#creating-translation-dictionary-files), i have not found no new translations.

Reply:
Ok. It seems the translation in Twig is out of scope of this PR.